tasks/util/watchified_bundle.js

```diff
@@ -6,7 +6,6 @@ var prettySize = require('prettysize');
 
 var constants = require('./constants');
 var common = require('./common');
-var compressAttributes = require('./compress_attributes');
```

Comment:
Nice that this just disappears. Opt-opt also scopes the transform better so it doesn't need to be applied globally 👍